src/OtpInterface.php

```diff
@@ -47,10 +47,11 @@ function totp($secret, $timecounter = null);
      * @param string  $secret  Base32 Secret String
      * @param integer $counter Counter
      * @param string  $key     User supplied key
+     * @param integer $counterwindow Size of the look-ahead window
      *
-     * @return boolean True if key is correct
+     * @return int|boolean the counter if key is correct else false
      */
-    function checkHotp($secret, $counter, $key);
+    function checkHotp($secret, $counter, $key, $counterwindow);
```

Comment:
Just for your own future reference: If you add something like this to an interface, make sure the signature is the same in both, right now this misses the part that this parameter is optional and the optional value.